Solutions/NXLogDnsLogs/Data Connectors/NXLogDnsLogs.json

```diff
@@ -0,0 +1,102 @@
+{
+    "id": "NXLogDnsLogs",
+    "title": "NXLog DNS Logs",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog DNS Logs data connector uses Event Tracing for Windows ([ETW](https://docs.microsoft.com/windows/apps/trace-processing/overview)) for collecting both Audit and Analytical DNS Server events. The [NXLog *im_etw* module](https://nxlog.co/documentation/nxlog-user-guide/im_etw.html) reads event tracing data directly for maximum efficiency, without the need to capture the event trace into an .etl file. This REST API connector can forward DNS Server events to Azure Sentinel in real time.",
+    "additionalRequirementBanner": "This data connector depends on parsers based on Kusto functions deployed with the Azure Sentinel Solution to work as expected. The [**NXLog_parsed_DNS_Server_ASim_view**](https://aka.ms/sentinel-nxlogdnslogs-parser) is designed to leverage Azure Sentinel's built-in DNS-related analytics capabilities.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "NXLog_parsed_DNS_Server_ASim_view",
+            "baseQuery": "NXLog_parsed_DNS_Server_ASim_view"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "DNS Server top 5 hostlookups",
+            "query": "NXLog_parsed_DNS_Server_ASim_view\n| summarize count() by Domain\n| take 5\n| render piechart title='Top 5 host lookups'"
+        },
+        {
+            "description" : "DNS Server Top 5 EventOriginalTypes (Event IDs)",
+            "query" : "NXLog_parsed_DNS_Server_ASim_view\n| extend EventID=strcat('Event ID ',trim_end('.0',tostring(EventOriginalType)))\n| summarize CountByEventID=count() by EventID\n| sort by CountByEventID\n| take 5\n| render piechart title='Top 5 EventOriginalTypes (Event IDs)'"
+        },
+        {
+            "description" : "DNS Server analytical events per second (EPS)",
+            "query" : "NXLog_parsed_DNS_Server_ASim_view\n| where EventEndTime >= todatetime('2021-09-17 03:07')\n| where EventEndTime <  todatetime('2021-09-18 03:14')\n| summarize EPS=count() by bin(EventEndTime, 1s)\n| render timechart title='DNS analytical events per second (EPS) - All event types'"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "DNS_Server_CL",
+            "lastDataReceivedQuery": "NXLog_parsed_DNS_Server_ASim_view            | summarize Time = max(TimeGenerated)            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "NXLog_parsed_DNS_Server_ASim_view | summarize LastLogReceived = max(TimeGenerated) | project IsConnected = LastLogReceived > ago(30d)"

```

Comment:
I always suggest writing the IsConnecedQuery on the source table. There is no reason to use the parsed one. i.e. DNS_Server_CL | .....

Also, a broader comment "DNS_Server_CL" is a very generic name and might lead to duplications. We may want to change that and use a different table name.

Reply:
Applied changes requested in commit [1c03252](https://github.com/Azure/Azure-Sentinel/pull/3056/commits/1c032529ea6917003a6cbe97ed125729803160c1).

Reply:
@oshezaf Thanks for the suggestions.

I changed the source table name from `DNS_Server_CL` to `NXLog_DNS_Server_CL` along with the references to it in the parser and data connector accordingly.